src/net.h

```diff
@@ -49,8 +49,8 @@ int hyper_setup_network(struct hyper_pod *pod);
 int hyper_cmd_setup_interface(char *json, int length);
 int hyper_cmd_setup_route(char *json, int length);
 int hyper_setup_dns(struct hyper_pod *pod);
-int hyper_get_type(int fd, uint32_t *type);
-int hyper_send_type(int fd, uint32_t type);
+int hyper_get_type(int fd, uint64_t *type);
+int hyper_send_type(int fd, uint64_t type);
```

Comment:
int64_t hyper_eventfd_receive(int fd);
int hyper_eventfd_send(int fd, uint64_t type);

and move the code to util.[ch]